packages/samples/angular/src/app/atomic-angular-page/atomic-angular-page.component.css

```diff
@@ -0,0 +1,163 @@
+atomic-color-facet[field='cat_color']::part(value-Black) {
```

Comment:
Pretty much everything here would be replaced by the layouts being currently worked on by @ThibodeauJF